### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by insign**
Typo

### CODE REVIEW

**Review by gz83 — APPROVED** [review]
LGTM, if we need to merge this PR, we also need Alex to give his LGTM and merge it

**Comment by gz83** [comment]
> LGTM, if we need to merge this PR, we also need Alex to give his LGTM and merge it

@insign @Alex313031

**Comment by Alex313031** [comment]
@insign Thanks. Merging.

**Pull request merged**
